db/migrate/20201216163610_add_import_count_and_failed_count_and_userid_to_imported_items.rb

```diff
@@ -0,0 +1,7 @@
+class AddImportCountAndFailedCountAndUseridToImportedItems < ActiveRecord::Migration[6.0]
+  def change
+    add_column :imported_items, :imported, :integer
+    add_column :imported_items, :failed, :integer
+    add_column :imported_items, :uploaded_by_user_id, :bigint
```

Comment:
Please create fk here and use user_id instead of uploaded_by_user_id.

Reply:
In a chat elsewhere it was suggested to name the record this way. Have actioned your request to change.